docs/tutorials/control/air-quality-fleet.md

```diff
@@ -13,48 +13,145 @@ level: "Intermediate"
+
+   In the next section, you'll add machines to the locations.
 
-1. For each sensing machine, connect the PM sensor to a USB port on the machine's SBC.
-2. Enable serial communication on each SBC.
+## Connect your machines to the Viam app
+
+With your organizational structure in place, let's add some machines:
+
+1. Connect the single-board computer you'll be shipping to Antonia to power.
+   If the computer does not already have a Viam-compatible operating system installed, follow the [Prepare your board steps in the Installation Guide](/get-started/installation/#prepare-your-board) to install a compatible operating system.
+   You _do not_ need to follow the "Install `viam-server`" section; you will do that in the next step!
+
+1. Enable serial communication so that the SBC can communicate with the air quality sensor.
    For example, if you are using a Raspberry Pi, SSH to it and [enable serial communication in `raspi-config`](/get-started/installation/prepare/rpi-setup/#enable-communication-protocols).
 
-3. Position your sensing machines in strategic locations, and connect them to power.
+1. Click **Antonia's Home** in the left navigation menu to navigate to that location's page.
+   In the **New machine** field near the top-right corner of the screen, type in a name for the machine, such as `Home Air Quality Sensor`, and click **Add machine**.
+
+1. You'll be taken to the machine details page and prompted to set up your machine part.
+   Click **View setup instructions**.
+   You can find these instructions later if you need them by clicking the part status indicator (which currently reads **Awaiting setup**).
+
+1. Follow the **Set up your machine part** instructions to install `viam-server` on the machine and connect it to the Viam app.
+   `viam-server` is the binary that runs on the single-board computer (SBC), providing functionality including sensor data collection and connection to the Viam app.
+
+   The setup page will indicate when the machine is successfully connected.
+
+1. If Antonia has more than one air sensing machine, add a new machine to her location and set it up in the same way.
+
+1. It's time to add some machines to the RobotsRUs location.
+   Connect one of the single-board computers you'll send to RobotsRUs' Oregon office to power.
+   Click **RobotsRUs** in the locations list.
+   Click the **Oregon Office** sub-location to navigate to its page.
+1. Add a new machine, naming it `air-sensor1` for example.
+1. Click **View setup instructions** and follow the steps on that page, just as you did with Antonia's machines.
+1. Repeat this process for each machine you're shipping to RobotsRUs' Oregon office, then navigate to the New York Office location and set up those machines.
```

Comment:
Maybe here we don't add steps but leave it as prose? Maybe that's going a bit into the direction of the feedback we got of holding hands a bit much when it's easy. Maybe this is enough?

```
This is how you set up one machine. If you are following along for the RobotsRUs business from our example, create additional machines in each sublocation, so in the `Oregon Office` location and in the `New York Office` location.
```